CONTRIBUTING.md

```diff
@@ -8,13 +8,16 @@
 - Make sure to read the "Restriction for New Feature(s)" section below.
 - When submitting a Pull Request, make sure that it is done to the same branch from which you first pulled, unless otherwise stated in the project's README, or if specifically instructed by maintainers of this project.
 - We don't require a fully descriptive PR, but please be clear on what added/removed and what the code does.
+## Language/Translation Contribution
+You can help us add or fixes localization on [Collapse on Crowdin!](https://crowdin.com/project/collapse-launcher)
+If you wish to add new language that isn't yet listed in the Crowdin project, make a discussion/issue either in GitHub or in Crowdin itself and we will get back to you at timely manner. 
 
 ## Tools Needed
 Below is the tools needed to contribute to this project.
 1. **Visual Studio 2022 (Any Edition - 17.4 or later)**
-2. .NET: [**.NET Core 7 SDK (7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
-3. WinUI 3: [**WinUI 3 (WindowsAppSDK 1.3.0-230331000 Stable Runtime)**](https://aka.ms/windowsappsdk/1.3/1.3.230331000/windowsappruntimeinstall-x64.exe)
-4. **Windows 10 SDK (10.0.19043.0) or Windows 11 SDK (10.0.22000.0)** via Visual Studio Installer
+2. **Windows 10 SDK (10.0.19043.0) or Windows 11 SDK (10.0.22000.0)** via Visual Studio Installer
+3. .NET: [**.NET Core 7 SDK (7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+4. WinUI 3: [**WinUI 3 (WindowsAppSDK 1.3.0-230331000 Stable Runtime)**](https://aka.ms/windowsappsdk/1.3/1.3.230331000/windowsappruntimeinstall-x64.exe)

```

Comment:
Instead of having the name of the tool required, we can just hyperlink directly to `.NET` & `WinUI 3`

Reply:
I think it's better to have them listed for devs that already have some of the components installed and just wanted to make sure they have the right version of it.

Reply:
I just feel that having the name and the version is redundant, so maybe we can find a compromise?

Reply:
we can do it something like this:
```md
## Tools Needed
Below is the tools needed to contribute to this project.
1. **Visual Studio 2022 (Any Edition - 17.4 or later)**
2. **Windows SDK (10.0.19043.0 or 10.0.22000.0 ONLY)** via Visual Studio Installer
3. .NET Core 7 SDK: [**(7.0.100 or later)**](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
4. WinUI 3: [**(WindowsAppSDK 1.3.0-230331000 Stable Runtime)**](https://aka.ms/windowsappsdk/1.3/1.3.230331000/windowsappruntimeinstall-x64.exe)
```

Reply:
we can also remove these completely from README so its much more cleaner there

Reply:
I think we should keep them since it'll be easier to see for people wanting to get themselves set up as fast as possible. The suggestion you provided above is good, so let's go with that.